_getting-started/intellij-track/testing-scala-in-intellij-with-scalatest.md

```diff
@@ -20,7 +20,7 @@ This assumes you know [how to build a project in IntelliJ](building-a-scala-proj
 1. Add the ScalaTest dependency:
     1. Add the ScalaTest dependency to your `build.sbt` file:
         ```
-        libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.5" % "test"
+        libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8"
```

Comment:
Why is `% "test"` removed?

Reply:
Well, I just copied sbt line from Scaladex. Actually, adding `% "test"` don't hurts.

Reply:
Yeah, http://www.scalatest.org/install suggests having `% "test"`.

Reply:
But what's the purpose of `% "test"`? The example work flawlessly with or without it.

Reply:
> Often a dependency is used by your test code (in `src/test/scala`, which is compiled by the `Test` configuration) but not your main code.
> 
> If you want a dependency to show up in the classpath only for the `Test` configuration and not the `Compile` configuration, add `% "test"` [...]

https://www.scala-sbt.org/release/docs/Library-Dependencies.html#Per-configuration+dependencies

Reply:
Thank you!